xbmc/GUIPassword.cpp

```diff
@@ -118,6 +109,26 @@ bool CGUIPassword::IsItemUnlocked(CFileItem* pItem, const std::string &strType)
   return true;
 }
 
+bool CGUIPassword::IsItemUnlocked(CFileItem* pItem, const std::string &strType)
+{
+  const std::string strLabel = pItem->GetLabel();
+  std::string strHeading = "";
+  if (pItem->m_bIsFolder)
+    strHeading = g_localizeStrings.Get(12325);
```

Comment:
These 5 digit numbers are opaque and reduce clarity. I usually add a comment with the string like [this](https://github.com/xbmc/xbmc/blob/master/xbmc/games/dialogs/GUIDialogSelectGameClient.cpp#L65). The string might change a little and get out of sync, but at least the meaning will be preserved.

Reply:
done